Makefile

```diff
@@ -472,4 +472,8 @@ dev-release:
 .PHONY: validate-release
 ## validate the operator by installing the releases
 validate-release:
+	$(Q)python3.7 -m venv $(OUTPUT_DIR)/venv3
+	$(Q)$(OUTPUT_DIR)/venv3/bin/pip install --upgrade setuptools
+	$(Q)$(OUTPUT_DIR)/venv3/bin/pip install --upgrade pip
+	$(Q)$(OUTPUT_DIR)/venv3/bin/pip install yq
```

Comment:
this installs always the latest version, so over the time, we could start using a newer `yq` version, potentially breaking builds without any obvious clue. To avoid that, we should fix the installed version on the latest currently available.

Reply:
Done. ce43a1d6db8cff3ff7f93d68f147d256607b8def